test/system/post_test.rb

```diff
@@ -0,0 +1,32 @@
+require "application_system_test_case"
+# https://guides.rubyonrails.org/testing.html#implementing-a-system-test
+
+class PostTest < ApplicationSystemTestCase
+
+  def setup
+    activate_authlogic
+  end
+
+  test 'posting from the editor' do
+    visit '/'
+
+    click_on 'Login'
+    fill_in("username-login", with: "Bob")
+    fill_in("password-signup", with: "secretive")
+    click_on "Log in"
+
+    visit '/post'
+
+    fill_in("Title", with: "My new post")
```

Comment:
fill_in(x, with: Y)
I can't get what is x? X is supposed to be class or id or what? I cannot find id with Title.

Reply:
X is ... i can't remember the term... maybe a `locator`? It's a Capybara term. I believe it can take the id, the name, or... not sure, and there's an order to it too: https://github.com/teamcapybara/capybara#the-dsl